main.go

```diff
@@ -42,6 +42,20 @@ workflows:
             pwd
 
             stepman audit --step-yml ./step.yml
+    - git::https://github.com/bitrise-steplib/steps-validate-json-schema.git@step-init:
```

Comment:
nit: This YAML string is getting bigger and bigger, why don't we `go:embed` it as a file into the binary?

Reply:
updated in: https://github.com/bitrise-steplib/steps-check/pull/14/commits/eac1ae41434bc7d5cd77fb6abbd6921a467870f4